WordPress/Classes/ViewRelated/Post/PostNoticeViewModel.swift

```diff
@@ -5,7 +5,13 @@ enum PostNoticeUserInfoKey {
 }
 
 struct PostNoticeViewModel {
-    let post: AbstractPost
+    private let post: AbstractPost
+    private let postCoordinator: PostCoordinator
+
+    init(post: AbstractPost, postCoordinator: PostCoordinator = PostCoordinator.shared) {
```

Comment:
Interesting. Why pass the postCoordinator as a parameter? Won't we always use PostCoordinator.shared?

Reply:
It's for dependency injection in the tests. In the tests, we pass a mocked `PostCoordinator`.